### ISSUE TITLE
Setup

### ISSUE THREAD

**Issue by jmagar** [SecretiveShell/MCP-Bridge#10]
How exactly do you set it up? Do you point mcp-bridge at my llm provider (openrouter) and then point open webui to the bridge or what?

**Comment by SecretiveShell** [comment]
Please refer to the how it works section for a diagram of how it works

https://github.com/SecretiveShell/MCP-Bridge/blob/5c34292eb9a51f9c90f77783ef79cd7e5d56ec19/README.md#how-does-it-work

Configure MCP-Bridge to point to your inference provider and then point your client at MCP Bridge.

OWUI --> MCP-Bridge --> Open Router

note that open router must have OAI style tool calling for this to work, and I have not tested it.

**Comment by gl2007** [comment]
So our application should call the MCP Bridge as a "drop in" for the OpenAI API? I am guessing scaling wouldn't be an issue since deployment via docker can take care of that.

**Comment by SecretiveShell** [comment]
> So our application should call the MCP Bridge as a "drop in" for the OpenAI API? I am guessing scaling wouldn't be an issue since deployment via docker can take care of that.

Yes, you can use MCP-Bridge as a drop in replacement for openai API

**Comment by buryhuang** [comment]
Does this work with docker type of mcp server. How does docker command works inside a container?

**Comment by SecretiveShell** [comment]
@buryhuang there are a few ways to go about this. You can:

- use docker in docker
- use docker out of docker (mount docker socket to container)

both methods will require you to install the docker CLI in the container build file, and you would specify an STDIO server with the `docker run` command

I have considered adding special support to run docker containers directly but I have not gotten around to it yet.

**Comment by buryhuang** [comment]
I used the following as config json:

```
{
   "inference_server": {
      "base_url": "https://api.openai.com/v1",
      "api_key": "my key"
   },
   "mcp_sererver": ...
}
```

I ran it on AWS ubuntu EC2 using command:
`uv run python mcp_bridge/main.py`

When I run it, I got the following error:

```
INFO:     xxxxx - "POST /chat/completions HTTP/1.1" 404 Not Found
INFO:     xxxxx - "POST /chat/completions HTTP/1.1" 404 Not Found

```
I could continue to debug it, but reporting it in case there is dump setup error I did.

**Comment by SecretiveShell** [comment]
closing due to inactivity
